lib/ui.ex

```diff
@@ -80,6 +81,26 @@ defmodule Ui do
     |> List.last
   end
 
+  defp get_board_size do
+    input = clean_input(IO.gets(""))
+    map_board_size_choice(input)
+  end
+
+  defp map_board_size_choice("1"), do: 3
```

Comment:
The function maps the users choice to the dimension of the board, that's why I used "map" here. And writing this just made me realise that the real function name should probably be this: `map_to_dimension`